unittests/test_state.py

```diff
@@ -0,0 +1,275 @@
+import unittest
+from unittest import mock
+from unittest.mock import Mock, MagicMock
+from control.state import OmapGatewayState
```

Comment:
In order to run these tests without a container in automation with Github actions, we need to mock rados on imports.

```suggestion
with mock.patch.dict("sys.modules", rados=mock.Mock()):
    from control.state import OmapGatewayState
```